pkg/traceutil/trace.go

```diff
@@ -102,13 +104,26 @@ func (t *Trace) InsertStep(at int, time time.Time, msg string, fields ...Field)
 	}
 }
 
+// SubTrace adds step to trace as a sublevel trace
+func (t *Trace) SubTrace(fields ...Field) {
```

Comment:
Does this also count towards the final "step count"?

Reply:
Also, can we provide a unit test or example to indicate the usage and expected behavior of this function?

Reply:
> Does this also count towards the final "step count"?

It should not count into `step_count`. I fixed it to `len(step)`


> Also, can we provide a unit test or example to indicate the usage and expected behavior of this function?

Added one test case in `TestLog`